.github/workflows/actions.yml

```diff
@@ -22,5 +22,9 @@ jobs:
     - name: Get Dependencies using Go Modules
       run: go mod download
 
-    - name: Build Coderunner
-      run: go build -o coderunner
+    - name: Build Server
+      run: go build -o runserver
+      
+    - name: Run and test server
+      run: ./runserver -port '8083' & go test ./...
```

Comment:
For now, this is okay. Enhance the test code to accept command-line arguments using the example below.
```go
package server

import (
  "flag"
  "testing"
)

var portNumber = flag.String("port", "8082", "Port number for server to listen on")

func TestSendZip(t *testing.T) {
  port := *portNumber
  // pass port along to sendZip().
}
```

The command to test would then look like `go test ./... -args -port=8082`

Reply:
Added to server_test.go